src/app/modules/table/table-data/table-data.component.html

```diff
@@ -37,6 +37,17 @@
         <fa-icon *ngIf="data.value.icon" [icon]="data.value.icon"></fa-icon>
     </app-popover>
 </div>
+
+<div *ngIf="data.value.el === 'tooltip'"
+    [id]="data.value.id"
+    class="{{data.key + ' ' + dataClass}}"
+    [ngClass]="data.value.attr ? data.value.attr : false"
+    [attr.title]="!data.value.title ? data.value.value : data.value.title"
+    tabindex="0"
+>   <app-tooltip *ngIf="data.value.tooltipModel" [tooltipModel] = 'data.value.tooltipModel'>
```

Comment:
Prefer `<app-tooltip` to be on the next line, indented. Rather than achieving "indentation" via spaces on the same line as the closing `>` from `<div`

Also, if we're already on 2 lines with `<app-tooltip`, we might as well do 1x attribute/param per line, like you did with `<div>`

Also also, don't need spaces on either side of the `=` after `[tooltipModel]` - since I'm fussing about whitespace already 😆